packages/robo/src/cli/utils/cloudflared.ts

```diff
@@ -200,3 +329,87 @@ function download(url: string, to: string, redirect = 0): Promise<string> {
 		request.end()
 	})
 }
+
+async function cloudflareRequest(endpoint: string, method: 'GET' | 'POST' | 'PUT' = 'GET', body: CloudflareBody | TunnelConfig | null = null): Promise<CloudflareResponse> {
+	const response = await fetch(`${CLOUDFLARE_API}${endpoint}`, {
+		method,
+		headers: {
+			'Authorization': `Bearer ${process.env.CLOUDFLARE_API_KEY}`,
+			'Content-Type': 'application/json',
+		},
+		body: body ? JSON.stringify(body) : null,
+	});
+
+	const data: CloudflareResponse = await response.json();
+
+	if (!response.ok || !data.success) {
+		cloudflareLogger.error(`Cloudflare API request failed: ${data.errors[0]?.message || 'Unknown error'}`);
+	} else {
+		cloudflareLogger.debug(`Cloudflare API request succeeded: ${endpoint}`)
+	}
+
+	return data;
+}
+
+async function updateTunnelConfig(id: string, accountId: string) {
+	const tunnelConfig: TunnelConfig = {
+		"config": {
+			"ingress": [
+				{
+					"hostname": `robo.${process.env.CLOUDFLARE_DOMAIN}`,
+					"service": "http://localhost:3000"
```

Comment:
Local service port should be `process.env.PORT` by default if that exists.

Reply:
Fixed. Port will default to 3000 if none is set in the .env file